Sources/SourceControl/GitRepository.swift

```diff
@@ -16,13 +16,13 @@ import func POSIX.getenv
 import enum POSIX.Error
 
 enum GitRepositoryProviderError: Swift.Error {
-    case gitCloneFailure(url: String, path: AbsolutePath)
+    case gitCloneFailure(url: String, path: AbsolutePath, errorOutput: String)
```

Comment:
We should catch this error in `Commands/Error.swift` so that it can be a bit pretty printed